core/src/main/java/com/google/errorprone/bugpatterns/javadoc/InvalidParam.java

```diff
@@ -101,6 +110,15 @@ public Description matchMethod(MethodTree methodTree, VisitorState state) {
     return Description.NO_MATCH;
   }
 
+  private static MethodTree getGeneratedConstructor(ClassTree classTree) {
+    return classTree.getMembers().stream()
```

Comment:
For records in some cases the generated constructor could not be found by using `ASTHelpers#isGeneratedConstructor`. That's why it's implemented with the `contentEquals("<init>")` check. 

Perhaps that is a bigger issue we should solve. However, I think that is out of scope for this PR.